hedera-mirror-rest/__tests__/specs/contracts-06-specific-id-not-found.spec.json

```
@@ -1,7 +1,22 @@
 {
   "description": "Contract api calls for a specific contract using contract id, not found",
-  "setup": {},
-  "url": ["/api/v1/contracts/0.0.6001", "/api/v1/contracts/60f3f640a8508fc6a86d45df051962668e1e8ac7"],
+  "setup": {
+    "contracts": [
+      {
+        "created_timestamp": "987654000123456",
+        "file_id": "5001",
+        "key": [1, 1, 1],
+        "num": "924569",
+        "timestamp_range": "[987654000123456,]"
```

```suggestion
        "timestamp_range": "[987654000123456,)"
```

in integrationDomainOps.js we insert a left closed right open range `[)` contract into `contract` table. Otherwise, we insert it into the `contract_history` table.